homeassistant/components/knx/__init__.py

```diff
@@ -316,34 +340,6 @@ def connection_config_tunneling(self):
             auto_reconnect=True,
         )
 
-    @callback
-    def async_create_exposures(self):
-        """Create exposures."""
```

Comment:
The iteration of `self.config[DOMAIN][CONF_KNX_EXPOSE]` is now done in `async_setup()` like all other platforms.
The rest moved to expose.py module. 
I have removed the `@callback` decorator there as I'm not sure why this is needed.

Reply:
If the function is every called from from `async_run_job` or `async_run_hass_job` or `async_track_state_change_event` etc, and isn't marked with `@callback` it will run in the executor instead of the event loop.